pennylane/tape/tapes/tape.py

```diff
@@ -96,6 +97,31 @@ def expand_tape(tape, depth=1, stop_at=None, expand_measurements=False):
 
     new_tape = tape.__class__()
 
+    if expand_measurements:
+        # Check for observables acting on the same wire. If present, observables must be
+        # qubit-wise commuting Pauli words. In this case, the tape is expanded with joint
+        # rotations and the observables updated to the computational basis. Note that this
+        # expansion acts on the original tape in place.
```

Comment:
Yes, not as far as I could see! I believe we only expand the tape from within the QNode's `construct()` method, and yes there we get rid of the original. This might be something to bear in mind for future though, so I left this comment.

Reply:
So the tape to be extended is treated a bit like a 'scratch' or a 'work tape' 😆